test/parallel/test-path-parse-format.js

```diff
@@ -218,8 +218,7 @@ function checkFormat(path, testCases) {
     }, common.expectsError({
       code: 'ERR_INVALID_ARG_TYPE',
       type: TypeError,
-      message: 'The "pathObject" argument must be of type Object. Received ' +
-               'type ' + typeName(pathObject)
+      message: `The "pathObject" argument must be of type Object. Received type ${typeName(pathObject)}`
```

Comment:
Does this get flagged as too long when you run `make jslint` (or `vcbuild jslint` on Windows)?

If so, maybe it can be split onto two lines?:

```js
message: 'The "pathObject" argument must be of type Object. ' +
  `Received type ${typeName(pathObject)}`;
```

It only gets rid of one of the concatenations rather than both that way, but I'm not sure there's a good way around it with the line length limit.